kb/2024-07-22/harvester_cluster_shutdown_and_restart.md

```diff
@@ -0,0 +1,576 @@
+| https://192.168.122.142:2379 | db04095b49eb5352 |   3.5.9 |   34 MB |     false |       true |        46 |    6538829 |            6538829 |        |
+| https://192.168.122.143:2379 | a21534d02463b347 |   3.5.9 |   34 MB |      true |      false |        46 |    6538829 |            6538829 |        |
++------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+```
+
+Run `kubectl delete pod -n kube-system etcd-name` command to delete the etcd pod on the `witness node` to trigger the pod replacement and leader re-election so that the `etcd` leader will be located on one of the `control-plane` nodes. Check the `etcd` leader again to make sure.
+
+```
++------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+|           ENDPOINT           |        ID        | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS |
++------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+| https://192.168.122.141:2379 | c70780b7862269c9 |   3.5.9 |   34 MB |      true |      false |        47 |    6538833 |            6538833 |        |
+| https://192.168.122.142:2379 | db04095b49eb5352 |   3.5.9 |   34 MB |     false |       true |        47 |    6538833 |            6538833 |        |
+| https://192.168.122.143:2379 | a21534d02463b347 |   3.5.9 |   34 MB |     false |       true |        47 |    6538833 |            6538833 |        |
++------------------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+```
+
+To now, the `etcd` has three running instances and the leader is located on the `control-plane` node.
+
+:::info important
+
+Write down the information of those nodes like name, ip, and the leader. Ideally give them a sequence like 1, 2, 3.
+
+:::
+
+Shutdown the two `IS LEADER == false` nodes one by one.
+
+```
+harvester-node-shell$ sudo -i shutdown
+```
+
+### 4.3 Shutdown the Last Control-plane Node
+
+Shutdown the last `IS LEADER == true` node. Remember its physical information for restarting it first in the steps below.
+
+```
+harvester-last-node-shell$ sudo -i shutdown
+```
+
+## 5. Restart
+
+If the Harvester cluster has been moved to a new location, or has been off for days, or your infrastructure has changes, check and test the [network stability](#12-keep-network-stability).
+
+### 5.1 Restart the Control-plane Nodes and the Witness Node
+
+The first step is to start those `etcd` located nodes one after another.
+
+#### Restart the Leader Control-plane Node
+
+Power on the [last shutdown node](#43-shutdown-the-last-control-plane-node) first. After about three minutes, continue the next step.
```

Comment:
> After about three minutes, continue the next step.

Does it make sense to use a time specification here? I think that the user's hardware is decisive in determining how long it takes. Is there not a possibility to execute a CLI command to check the status?